packages/theme-chalk/src/checkbox-button.scss

```
@@ -70,6 +71,7 @@
       color: getCssVar('checkbox-button-checked-text-color');
       background-color: getCssVar('checkbox-button-checked-bg-color');
       border-color: getCssVar('checkbox-button-checked-border-color');
+      outline-color: getCssVar('checkbox-button-checked-border-color');
```

Stylelint (17.12.0)

[error] Expected "getCssVar" to be "getcssvar" (function-name-case)

(function-name-case)